docs/source/terminology.rst

```diff
@@ -35,16 +35,36 @@ local (map) frame
   point in time, the transform between this and the global map frame may change,
   but the transform between this and all other frames does not change.
 
+submap frame
+  There is a separate fixed submap frame for each submap.
+
 tracking frame
-  The frame in which sensor data is expressed.
+  The frame in which sensor data is expressed. It is not fixed, i.e. changes
+  over time. It is also different for different trajectories.
+
+gravity-aligned frame
+  Only used in 2D. A frame colocated with the tracking frame but with a
+  different orientation which is approximately aligned with gravity, i.e. the
+  gravitational acceleration vector points approximately in the -z direction. No
+  assumption about yaw (rotation around the z axis between this and the tracking
+  frame) should be made. A different gravity-aligned frame is used for different
+  trajectory nodes.
```

Comment:
Maybe explicitly call out that the yaw relationship between tracking and gravity aligned can changed arbitrarily between e.g. consecutive. Essentially in time gravity-aligned frames have no yaw relationship to one another.

Reply:
> between e.g. consecutive.

Have you accidentally a word?